tests/src/test/scala/ch/epfl/bluebrain/nexus/tests/kg/files/RemoteStorageSpec.scala

```diff
@@ -61,60 +54,26 @@ class RemoteStorageSpec extends StorageSpec with CopyFilesSpec {
     )
 
   override def createStorages(projectRef: String, storId: String, storName: String): IO[Assertion] = {
-    val payload       = jsonContentOf(
-      "kg/storages/remote-disk.json",
-      "endpoint" -> externalEndpoint,
-      "read"     -> "resources/read",
-      "write"    -> "files/write",
-      "folder"   -> remoteFolder,
-      "id"       -> storId
-    )
     val storageId2    = s"${storId}2"
     val storage2Read  = s"$storName/read"
     val storage2Write = s"$storName/write"
-    val payload2      = jsonContentOf(
-      "kg/storages/remote-disk.json",
-      "endpoint" -> externalEndpoint,
-      "read"     -> storage2Read,
-      "write"    -> storage2Write,
-      "folder"   -> remoteFolder,
-      "id"       -> storageId2
-    )
+
+    val expectedStorage          = storageResponse(projectRef, storId, "resources/read", "files/write")
+    val expectedStorageSource    =
+      jsonContentOf(
+        "kg/storages/storage-source.json",
+        "folder"      -> remoteFolder,
+        "storageBase" -> StoragesDsl.StorageServiceBaseUrl,
+        "id"          -> storId
+      )
+    val expectedStorageWithPerms = storageResponse(projectRef, storageId2, storage2Read, storage2Write)
 
     for {
-      _ <- deltaClient.post[Json](s"/storages/$projectRef", payload, Coyote) { (json, response) =>
-             if (response.status != StatusCodes.Created) {
-               fail(s"Unexpected status '${response.status}', response:\n${json.spaces2}")
-             } else succeed
-           }
-      _ <- deltaClient.get[Json](s"/storages/$projectRef/nxv:$storId", Coyote) { (json, response) =>
-             val expected = storageResponse(projectRef, storId, "resources/read", "files/write")
-             filterMetadataKeys(json) should equalIgnoreArrayOrder(expected)
-             response.status shouldEqual StatusCodes.OK
-           }
-      _ <- deltaClient.get[Json](s"/storages/$projectRef/nxv:$storId/source", Coyote) { (json, response) =>
-             response.status shouldEqual StatusCodes.OK
-             val expected = jsonContentOf(
-               "kg/storages/storage-source.json",
-               "folder"      -> remoteFolder,
-               "storageBase" -> externalEndpoint,
-               "id"          -> storId
-             )
-             filterKey("credentials")(json) should equalIgnoreArrayOrder(expected)
-
-           }
-      _ <- permissionDsl.addPermissions(
-             Permission(storName, "read"),
-             Permission(storName, "write")
-           )
```

Comment:
I removed these calls for each storage type tests and nothing failed... So I guess they're not needed, but I don't know why?

Reply:
Maybe a bug ?
When you create a storage, you can define custom permissions and those permissions need to have been previoully defined.

Reply:
This was the issue - I nuked the DB and it correctly failed afterwards. Fixed now. I have to remember there's no cleanup so the tests aren't independent 😅  maybe with generated data everywhere they would be unlikely to overlap